x-pack/plugins/ingest_pipelines/server/lib/mapper.ts

```diff
@@ -76,10 +76,17 @@ function parseAndValidate(file: string) {
     return [...acc, header];
   }, []);
 
-  if (missingHeaders.length > 0) {
+  if (missingHeaders.length === 1) {
+    throw new Error(
+      i18n.translate('xpack.ingestPipelines.mapToIngestPipeline.error.missingSingleHeader', {
+        defaultMessage: 'Missing required headers: Include {missing} header in the CSV file.',
+        values: { missing: missingHeaders[0] },
+      })
+    );
+  } else if (missingHeaders.length > 1) {
```

Comment:
What do you think about simplifying this by using the [pluralization feature](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#pluralization) from `kbn-i18n`?

```
if (missingHeaders.length > 0) {
  throw new Error(
      i18n.translate('xpack.ingestPipelines.mapToIngestPipeline.error.missingHeaders', {
      defaultMessage: 'Missing required headers: Include {missing} {missingHeadersCount, plural, one {header} other {headers} in the CSV file.',
      values: {
        missingHeaders: missingHeaders.join(', '),
        missingHeadersCount: missingHeaders.length,
      },
    })
  );
}
```

Reply:
I love that! I did not know that existed. Will update